homeassistant/components/smarty/sensor.py

```diff
@@ -64,11 +64,15 @@ class SmartySensorDescription(SensorEntityDescription):
     SmartySensorDescription(
         key="supply_fan_speed",
         translation_key="supply_fan_speed",
+        device_class=SensorDeviceClass.SPEED,
+        native_unit_of_measurement=REVOLUTIONS_PER_MINUTE,
         value_fn=lambda smarty: smarty.supply_fan_speed,
     ),
     SmartySensorDescription(
         key="extract_fan_speed",
         translation_key="extract_fan_speed",
+        device_class=SensorDeviceClass.SPEED,
```

Comment:
Revolutions per minute is not a good unit of measurement for device class speed. https://developers.home-assistant.io/docs/core/entity/sensor#available-device-classes So please remove the device class